src/scheduler/parse.c

```diff
@@ -560,7 +561,7 @@ parse_config(char *fname)
 											conf.preempt_order[i].order[j] = PREEMPT_METHOD_SUSPEND;
 											break;
 										case 'C':
-											conf.preempt_order[i].order[j] =PREEMPT_METHOD_CHECKPOINT;
+											conf.preempt_order[i].order[j] = PREEMPT_METHOD_CHECKPOINT;
 											break;
 										case 'R':
 											conf.preempt_order[i].order[j] = PREEMPT_METHOD_REQUEUE;
```

Comment:
Don't parse the old parameters.  Just set the obsolete[] array.  It has two elements.  The first element is what is being deprecated.  The second element is how you set it now.  Look for other examples of how it is used.

Reply:
done.